index.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="ja">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>test</title>
+  <script src="https://cdn.jsdelivr.net/npm/vue@2/dist/vue.js"></script>
+</head>
+
+<body>
+  <div id="app">
+    <h1>TODO LIST</h1>
+    <ul>
+      <li v-for="todo in todos" :key="todo.id">
+        <p>
+          {{ todo.text }}
+          <button @click="deleteTodo(todo.id)">削除</button>
+          <button @click="tryToEditTodo(todo.id)">編集</button>
+        </p>
+      </li>
+    </ul>
+    <h2>新規作成する</h2>
+    <input v-model="addText">
+    <button @click="addTodo()">TODO LISTへ追加</button><br>
+    <h2>編集する</h2>
+    <input v-model="editText">
+    <button @click="editTodo()">更新</button>
+  </div>
+
+  <script>
+    new Vue({
+      el: '#app',
+      data: function () {
+        return {
+          todos: [],
+          addText: '',
+          editId: '',
+          editText: ''
+        }
+      },
+      mounted () {
+        if (window.localStorage.getItem('todos')) {
+          this.todos = JSON.parse(window.localStorage.getItem('todos'))
+        }
+      },
+      methods: {
+        getMaxId () {
+          if (this.todos.length !== 0) {
+            const ids = this.todos.map(item => item.id)
+            return Math.max(...ids)
```

Comment:
`reduce()` を使っても書けます。直しても直さなくても大丈夫です。https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/max#getting_the_maximum_element_of_an_array
```suggestion
            return this.todos.reduce((todo1, todo2) => Math.max(todo1.id, todo2.id))
```